README.md

```diff
@@ -74,6 +74,11 @@ launch a build on that branch, and then optionally switch to different
 working branch to make changes without affecting any currently executing
 builds.
 
+
```

Comment:
I am not certain if this is the best place for this. It makes reference to `OPENXT_BRANCH` before it has been defined. I think here a short description that the `repo start` command is used for creating a working branch.

Reply:
@dpsmith  maybe not defined but it is referenced in the paragraph above it and it is the paragraph describing the  repo tool to create branches.

Reply:
Okay